rbac/common/task/task_transaction.proto

```diff
@@ -62,21 +47,6 @@ message ProposeAddTaskAdmin {
 }
 
 
-message ProposeRemoveTaskAdmin {
```

Comment:
assuming this and the `ProposeRemoveTaskOwner` messages you removed had never been implemented?

Reply:
yes, never implemented